modules/maven-plugin-tester/pom.xml

```diff
@@ -25,14 +25,24 @@
                 <groupId>com.smartbear.readyapi</groupId>
                 <artifactId>readyapi4j-maven-plugin</artifactId>
                 <version>${project.version}</version>
-                <configuration>
-                    <username>demoUser</username>
-                    <password>demoPassword</password>
-                    <server>http://testserver.readyapi.io:8080</server>
-                </configuration>
                 <executions>
                     <execution>
-                        <id>run</id>
+                        <id>RunLocally</id>
+                        <configuration>
+                            <runXmlProjects>false</runXmlProjects>
```

Comment:
ok - i was thinking that the user shouldn't have to configure anything for local execution - if they specify projects to execute the build should simply fail as you indicate - and if they add testserver credentials the plugin will automatically use remote execution